apps/src/EditorAnnotator.js

```diff
@@ -1109,7 +1109,7 @@ export default class EditorAnnotator {
    * Removes annotations of a particular type.
    */
   static clearAnnotations(logLevel = 'INFO') {
-    EditorAnnotator.annotator().clearAnnotations(logLevel);
+    EditorAnnotator.annotator()?.clearAnnotations(logLevel);
```

Comment:
Adding these null checks to get around a console error we were seeing when the annotations race the loading of the editor.